odoo/addons/base/tests/test_configmanager.py

```diff
@@ -340,7 +336,7 @@ def test_04_odoo16_config_file(self):
             'email_from': '',
             'geoip_city_db': '/usr/share/GeoIP/GeoLite2-City.mmdb',
             'http_enable': True,
-            'http_interface': '0.0.0.0',
+            'http_interface': '',
```

Comment:
should be 127.0.0.1

Reply:
also 16.0 is not supported anymore, we should go in 17.0, generate the default config, and use it in this test (that should be renamed aswell)